### ISSUE TITLE
obsolete `libjpeg8-dev` package referenced in README.md

### ISSUE THREAD

**Issue by richterdavid** [openzim/python-scraperlib#152]
`libjpeg8-dev` referenced in the [https://github.com/openzim/python-scraperlib/blob/7d498319baadba715316c15cf9857ff2f6974a00/README.md?plain=1#L44](README.md) is obsolete, and presumably should be replaced.

```bash
sudo apt install libmagic1 wget ffmpeg \
    libtiff5-dev libjpeg8-dev libopenjp2-7-dev zlib1g-dev \
    libfreetype6-dev liblcms2-dev libwebp-dev tcl8.6-dev tk8.6-dev python3-tk \
    libharfbuzz-dev libfribidi-dev libxcb1-dev gifsicle
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package libjpeg8-dev is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  libjpeg62-turbo-dev

E: Package 'libjpeg8-dev' has no installation candidate
```

**Comment by benoit74** [comment]
Thank you. 

It is not clear to me how mandatory this lib is anyway and all others mentioned in the README (especially the "dev" headers).

Looks like Debian decided to replace this lib some time ago, but it is not 100% equivalent: https://unix.stackexchange.com/a/727617

**Comment by rgaudin** [comment]
Required for Pillow's image conversions. I suppose the turbo one can be used but it should be checked.

**Comment by benoit74** [comment]
This is needed only for building from source, are we doing this?

**Comment by benoit74** [comment]
Agreed, in most situations a new developer on scraperlib probably does not need most of the mentioned requirements since he will probably be able to use the wheels. And even if he doesn't, he might well not be blocked in his development because not concerned by this part of the codebase.

**Comment by its-me-maady** [comment]
Hi, I'd like to work on this.

Plan: replace `libjpeg8-dev` with `libjpeg-dev` (virtual metapackage, resolves correctly on both Debian and Ubuntu) and add a short note that the `-dev` headers are only needed when building Pillow from source.

Does that approach work, or is there a preferred replacement?

**Comment by its-me-maady** [comment]
This issue along with #153 has been inactive for a while — combining this with #153 in a single PR since both are README fixes. Submitting shortly.
